src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs

```diff
@@ -35,7 +35,7 @@ internal FilePickerSelectedFilesArray(StorageFile[] items)
 		public int Count
 		{
 			get => _items.Count;
-			set => throw new InvalidOperationException();
+			//set => throw new InvalidOperationException();
```

Comment:
It's probably a breaking change to remove this particular setter. Is it available on Windows? If not, it's likely to be an acceptable breaking change for 5.0.

Reply:
According to [this](https://github.com/unoplatform/uno/issues/6885#issuecomment-911658880) comment it should be acceptable.

Reply:
Right! You'll likely need to add a exclusion here: https://github.com/unoplatform/uno/blob/master/build/PackageDiffIgnore.xml

Reply:
Done

Reply:
You can also remove the commented line, it's not useful anymore. Thanks!

Reply:
Removed.

Reply:
@jeromelaban can this conversation be resolved?